### PR TITLE
feat: added button component

### PR DESCRIPTION
**Pull request by pradipgharge**

### CODE REVIEW

**Review by manish-GH — APPROVED** [review]
Your design looks amazing, nice work.
You could have used just one btn-disabled instead of btn-solid-disabled and btn-outline disabled

**Comment by pradipgharge** [comment]
> Your design looks amazing, nice work. You could have used just one btn-disabled instead of btn-solid-disabled and btn-outline disabled

Thanks, bro. Actually btn-outline-disabled has text colour different from btn-solid-disabled, that's why had to create separately. Also to maintain consistency with class names.

File: pages/components/Buttons/button.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+.btn{
+    cursor: pointer;
+    border-radius: 10px;
+    border: none;
+    background-color: transparent;
+    font-size: 1rem;
+    padding: 0.8rem 1rem;
+}
+
+
```
Comment (shangkaul) [inline pages/components/Buttons/button.css:28]: You can add an :active pseudo class to button for giving user a slight feedback on btw click - (see translateY, button zoom on click) or a simple color change also works. 

This is my POV, just a suggestion, btw the component looks great!

**Review by shangkaul — APPROVED** [review]
Yep looks good, ready to merge!

File: pages/components/Buttons/button.css (hunk 1 above)
Comment (pradipgharge) [reply pages/components/Buttons/button.css:28]: Hey Shivang, thanks for the suggestion, I added the feature.

**Pull request merged**